### PR TITLE
add western eastern 2022 lidar to elevation page

### PR DESCRIPTION
**Pull request by rkelson**
new lidar project needs to be added to the lidar page

### CODE REVIEW

File: src/pages/products/sgid/elevation/lidar.astro [hunk 1]
```diff
@@ -653,6 +674,22 @@ const projectAssets = {
...
     </p>
   </Section>
+  <Section {...page.section[28]}>
+    <a href={westernEastern2022Map.src} slot="image">
+      <Image src={westernEastern2022Map} width="400" alt="A map with project boundary" class="w-full max-w-[400px]" /></a
+    >
+    <p>
+      During 2022 UGRC, the Utah Division of Emergency Management, the Natural Resource Conservation Service (NRCS), the
+      U.S. Forest Service (USFS), and the Federal Emergency Management Agency (FEMA) acquired ~19,466 square miles of 2
+      and 8 points per meter Quality Level 1 & 2 Lidar of portions Utah. The 0.5 and 1 meter resolution bare earth DEMs
```
Comment (steveoh) [inline src/pages/products/sgid/elevation/lidar.astro:684]: **thought**: the `of portions Utah.` part of the sentence reads funny. Does it need another of? 

> yada yada acquired Lidar of portions of Utah?

Comment (rkelson) [reply src/pages/products/sgid/elevation/lidar.astro:684]: good catch and I see some other changes needed

**Comment by rkelson** [comment]
@steveoh the image in the deploy is fuzzy. Will that be fixed when it is published? I used the same methodology as in the past to make this image

**Comment by rkelson** [comment]
I think this is ready to go

**Review by steveoh — APPROVED** [review]

**Pull request merged**